CMakeLists.txt

```diff
@@ -20,6 +20,16 @@ include(GNUInstallDirs)
 list(APPEND CMAKE_MODULE_PATH /usr/local/share/cmake/Modules)
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
 
+# Include Ogre cmake file on Arch linux
+execute_process(
+  COMMAND uname --kernel-release
+  OUTPUT_VARIABLE KERNEL
+  )
+if(KERNEL MATCHES "ARCH")
+  list(APPEND CMAKE_MODULE_PATH "/usr/lib/OGRE/OGRE/cmake")
```

Comment:
I don't think this works for me on Arch.

I actually have to use legacy OGRE 1.9 on Arch Linux which is installed in:
`/opt/OGRE-1.9/lib/pkgconfig`.

How did you install all the dependencies?

Reply:
It is similar to me, I had to install a legacy Ogre in the same path as you. You need also to update the `LD_LIBRARY_PATH`. However, even with that, cmake is not able to find Ogre cmake files since it is not in the default path. Therefore I had to indicate for cmake where to find these files.
@julianoes How does you cmake find Ogre?? I think you updated your `PKG_CONFIG_PATH`, Right?

Reply:
Yes, I manually set `PKG_CONFIG_PATH` so that it finds the legacy OGRE.
```
PKG_CONFIG_PATH="/opt/OGRE-1.9/lib/pkgconfig
```